lib/options-parser.js

```diff
@@ -44,6 +46,7 @@ function parseOptions(argsOriginal) {
     parsed.url = args[0];
     parsed.css = args[1];
 
+    // this is problematic...
```

Comment:
That seems way premature. Don't remove things if you don't know IF they actually pose a problem, as this test intend to serve the purpose of early warning :-) As `critical` is just using the file paths, they must themselves be using some kind of server (like we are doing in the tests) to serve the files. Not rocket science :-)

```
var app = module.exports = express();
app.use(express.static(__dirname ));
app.listen(port)
```